performance-test/src/gatling/java/org/opensearch/dataprepper/test/performance/tools/Templates.java

```diff
@@ -39,10 +40,24 @@ public static Function<Session, String> apacheCommonLogTemplate(final int batchS
         };
     }
 
+    public static Function<Session, String> userAgent(final int batchSize) {
+        return session -> {
+            final List<String> logs = IntStream.range(0, batchSize)
+                    .mapToObj(i -> "{\"log\": \"" + userAgent() + "\"}")
+                    .collect(Collectors.toList());
+            final String logArray = String.join(",", logs);
+            return "[" + logArray + "]";
```

Comment:
Why not use `Collectors.joining(",", "[","]")` here?

Reply:
I copied what was already in place here:

https://github.com/opensearch-project/data-prepper/blob/720ca89cb1cdd930c6f5c7a8d6aaffb73988e594/performance-test/src/gatling/java/org/opensearch/dataprepper/test/performance/tools/Templates.java#L33-L41

But, that is a good idea.